app/views/layouts/application.html.erb

```diff
@@ -31,15 +31,23 @@
         <div class="top-nav col s10">
           <h1 class="page-title"><%= @page_title %></h1>
           <div class="avatar-corner">
-            <p>Username</p>
+            <p><%= current_username %></p>
```

Comment:
How do we update this via ajax? Successful login, whether via Register or via Login should update this to display the user's name. Not a bug. Missing requirement. Needs to be added to Gherkin.

Reply:
I think that a more robust JSON object needs to be passed back upon a successful login/register via popup. In that JSON object would hold our username and then I can live update that as well. I was considering it, but I also didn't want to bloat this object up just because I couldn't immediately think of anything else.